File: DumbChecker/dumbChecker.cpp
#include <iostream>

int main()
{
    try 
    {
        int luckyNumber;
        std::cout << "Enter your lucky number : ";
        std::cin >> luckyNumber;

        srand(time(0));
        int randomNumber = rand() % 100;
        std::cout << "You are " << randomNumber << "% dumb!" << std::endl;
    }
    catch (std::exception e) 
    {
        std::cout << "Run again and enter a number dumbA**" << std::endl;
    }
}

File: DumbChecker/dumbChecker.py
import random

try:
    lucky_number = input("Enter your lucky number: ")
    lucky_number = int(lucky_number)
    random_number = round(random.random() * 100 + 10)
    print(f"You are {random_number}% dumb!")
except ValueError:
    print("Run again and enter a number dumbA**")
